### DISCUSSION TITLE
framer-motion version with Remix.run

### DISCUSSION THREAD

**Discussion by humphd** [chakra-ui/chakra-ui#7320] [Q&A]
In the docs for installing Chakra-UI with Remix.run, it says to use `framer-motion@^6`.  However, `framer-motion` is now at 9.x.x, and I'm wondering about this long term.  What are the reasons for using 6 vs. 9?  Is there a plan to be able to upgrade it, or remove it if not?

**Comment by junbach** [comment]
same question, framer-motion is v10 now

**Comment by humphd** [comment]
I tried updating `framer-motion` to `v10.11.2` and rebuilt my Remix app.  Everything *appears* to work.  Is there something I'm missing about why https://chakra-ui.com/getting-started/remix-guide is so specific about using `framer-motion@^6`?

I'd love to not leave this dep 4 major versions out of date if possible.
